.github/CODEOWNERS

```diff
@@ -684,7 +684,7 @@
 /sdk/quantum/Azure.Quantum.Jobs/                                   @xfield
 
 # ServiceLabel: %Recovery Services Backup %Service Attention
-/sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/ @pvrk @Daya-Patil
+/sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/ @Daya-Patil
```

Comment:
The `Microsoft.Azure.Management.*` packages are no longer supported.  Do we need this  entry?

Reply:
The partner team is asking for this to be updated and they do see some issues occasionally.

Reply:
Unsupported means unsupported.   Any issue that comes through for a `Microsoft.Azure.Management` package is closed at triage.   It'll never be routed to the service team.    They should only get issues from the `Azure.ResourceManager` libraries.